libraries/stdlib/src/kotlin/io/FileReadWrite.kt

```diff
@@ -129,19 +129,16 @@ public fun File.forEachBlock(action: (buffer: ByteArray, bytesRead: Int) -> Unit
  */
 public fun File.forEachBlock(blockSize: Int, action: (buffer: ByteArray, bytesRead: Int) -> Unit): Unit {
     val arr = ByteArray(blockSize.coerceAtLeast(MINIMUM_BLOCK_SIZE))
-    val fis = FileInputStream(this)
 
-    try {
+    inputStream().use {
         do {
-            val size = fis.read(arr)
+            val size = it.read(arr)
```

Comment:
Better to add a more descriptive name instead of `it`, as it is inside a multiline lambda, for example `stream` or `input`.

Reply:
Will change it to `input`.

Reply:
Agree with @ilya-g, but also wonder if it would be better when we would use in line 56 `length` or eventually `size` instead of mystical `it`.